conda-forge.yml

```diff
@@ -3,6 +3,7 @@ appveyor:
 compiler_stack: comp7
 max_py_ver: '37'
 max_r_ver: '35'
+provider: {linux_aarch64: azure, linux_ppc64le: azure, win: azure}
```

Comment:
```suggestion
provider: {linux_aarch64: azure, linux_ppc64le: azure, win: azure}
```

Reply:
??? why didn't rerendering work...

Reply:
I had an extra "provider" line at the bottom of the conda-forge.yml file that was clobbering the one you put in.